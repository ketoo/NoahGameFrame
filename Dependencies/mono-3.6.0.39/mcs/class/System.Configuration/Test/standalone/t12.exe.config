<configuration>
	<configSections>
		<section name="customSection" type="CustomSection, t12" />
	</configSections>

	<customSection longSetting="10" />
</configuration>