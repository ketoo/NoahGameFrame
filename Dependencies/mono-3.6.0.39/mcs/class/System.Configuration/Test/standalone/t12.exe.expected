1
2
Exception raised: System.Exception
